src/realm.js

```diff
@@ -231,6 +233,9 @@ export class Realm {
 
   globalSymbolRegistry: Array<{ $Key: string, $Symbol: SymbolValue }>;
 
+  attachedDebugger: Debugger | void;
```

Comment:
There are two modes for debugger: attach, launch. I would call this "debuggerInstance" instead so not to be confused with launch/attach action mode.